### ISSUE TITLE
fromScreenLocation/toScreenLocation require flipped y

### ISSUE THREAD

**Issue by ljbade** [mapbox/mapbox-gl-native#2519]
Currently where we use `fromScreenLocation` or `toScreenLocation` the calling code has to flip Y direction between Android's and GL's coordinate space. This is confusing for end developers who will expect to just pass a normal Android X, Y point.

I will hit this in the PR for #2507

**Comment by bleege** [comment]
This actually was merged as part of #2529 which makes it `android-v2.1.0`.

**Comment by bleege** [comment]
We should add a simple "Press Map To Add Marker" test activity to the TestApp so that we can better visualize this.

**Comment by tobrun** [comment]
If @ljbade or @bleege haven't picked that up, I will in the morning. 
I was also thinking about simplifying MainActivity and moving some things in separate features.

**Comment by bleege** [comment]
@tobrun Already on it.  ;-)  Thanks though!

**Comment by tobrun** [comment]
@bleege cool, will add a separate issue for those other things ;)

**Comment by bleege** [comment]
Here's what the test activity looks like.

![device-2015-10-08-162311](https://cloud.githubusercontent.com/assets/989928/10380752/5abc70f2-6ddb-11e5-9769-86ae658c966b.png)

**Comment by tobrun** [comment]
Should the default location of attribution icon be to the left or to the
right? It seems more natural if it would be the same as in iOS..

**Comment by bleege** [comment]
> Should the default location of attribution icon be to the left or to the right?

We can certainly move it to the right in this particular case.  Ever since the FAB was announced by Google, [I'm hesitant to default anything to the lower right corner](https://github.com/mapbox/mapbox-gl-native/pull/2220#issuecomment-138598604) in portrait mode.

Incidentally, I had just rebased / merged this new code into `master` via #2569 before I saw this comment.  I'm going to close this issue for now as the tech to complete this feature is complete.  That said we can open a new ticket to discuss the attribution location if you'd like.
